
 Workaround "argument-dependent lookup" problems with gcc versions >= 4.7. The proper solution would be to fix the uClibc++ header files.

--- Rules.mak
+++ Rules.mak
@@ -195,7 +195,7 @@
 CPU_CFLAGS=$(call qstrip,$(CPU_CFLAGS-y))
 
 # Some nice CFLAGS to work with
-GEN_CFLAGS:=-fno-builtin
+GEN_CFLAGS:=-fno-builtin -fpermissive
 CFLAGS:=$(XWARNINGS) $(CPU_CFLAGS) -ansi
 
 LDFLAGS-$(LIBNAME).so:=-Wl,--warn-common -Wl,--warn-once -Wl,-z,combreloc -Wl,-z,defs
